BrainPortal/config/oidc.yml.erb

```diff
@@ -0,0 +1,19 @@
+Keycloak:
+  # A OpenID provider can be enabled
+  enabled:                            true
+  # Configuration of the OpenID provider
+  authorize_uri:                      "http://localhost:8080/realms/.../protocol/openid-connect/auth"
```

Comment:
What are the `...` in the path? Ar they things I'm supposed to replace/substitute? What is supposed to go there?